doerffelbot/main.py

```diff
@@ -9,8 +9,6 @@
 
 
 TOKEN = conf.TOKEN  # Token of @doerffelbot
-old_TOKEN = conf.old_TOKEN  # Token of @johannstodobot
-my_chat_id = conf.my_chat_id  # my own telegram-'chat id'
```

Comment:
Don't remove that line!

Reply:
Why not? That variable is not used anywhere in that file

Reply:
It's not in the scope of this issue :)